src/hearings/containers/request-hearing/hearing-view-edit-summary/hearing-view-edit-summary.component.ts

```diff
@@ -64,7 +66,7 @@ export class HearingViewEditSummaryComponent extends RequestHearingPageFlow impl
   }
 
   public setPropertiesUpdatedOnPageVisit(): void {
-    this.hearingValuesSubscription = this.hearingStore.select(fromHearingStore.getHearingValues).pipe(take(1)).subscribe((hearingValues) => {
+    this.hearingValuesSubscription = this.hearingStore.select(fromHearingStore.getHearingValues).subscribe((hearingValues) => {
```

Comment:
@johnbenjamin-hmcts - now that `this.hearingValuesSubscription` has had the `take(1)` removed, is this `Subscription` being unsubscribed somewhere?

Reply:
@Daniel-Lam - the `this.hearingValuesSubscription` is unsubscribed in the `ngOnDestroy()` method.

The `take(1)` in this case was not returning the latest results, whereas returning the previous results from the store, so had to remove it.